backend/src/constants/ExampleUserList.ts

```diff
@@ -0,0 +1,31 @@
+const ExampleUserList = new Array(20);
+interface User {
+  userID: string;
+  username: string;
+  email: string;
+  about: string;
+  websiteUrl: string;
+  profilePicture: string;
+  coverPicture: string;
+}
+const createUsersArray = (arr: User[]) => {
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
suggestion: replace for loop with map:
const users = new Array(20).fill('').map((_, index) => ({id: index, username: `exampleName${index}`}))